eras/shelley-ma/impl/src/Cardano/Ledger/Allegra/Translation.hs

```diff
@@ -61,7 +69,11 @@ instance Crypto c => TranslateEra (AllegraEra c) NewEpochState where
           nesBcur = nesBcur nes,
           nesEs = translateEra' ctxt $ LS.returnRedeemAddrsToReserves . nesEs $ nes,
```

Comment:
The use of `returnRedeemAddrsToReserves` here might be confusing to future readers. This function both removes the AVVM addresses from the UTxO and also increases the reserves balance by the corresponding amount. Here, however, the first operation is a no-op since we (the informed) know that we have already removed the AVVM, and so `returnRedeemAddrsToReserves` is only being used to increase the reserves. At the very least, maybe we could add a comment?

Reply:
We haven't removed the AVVM addresses! On the Byron/Shelley border, we don't remove them, we just take note of them. They are removed, and the pots adjusted, on the Shelley/Allegra border, exactly the same as before.

Reply:
ah!!